tools/packaging/packaging.psm1

```diff
@@ -850,33 +705,41 @@ function New-UnixPackage {
                     Move-Item $hack_dest $symlink_dest
                 }
             }
-            if ($AfterInstallScript) {
-                Remove-Item -erroraction 'silentlycontinue' $AfterInstallScript
+            if ($AfterScriptInfo.AfterInstallScript) {
+                Remove-Item -erroraction 'silentlycontinue' $AfterScriptInfo.AfterInstallScript
             }
-            if ($AfterRemoveScript) {
-                Remove-Item -erroraction 'silentlycontinue' $AfterRemoveScript
+            if ($AfterScriptInfo.AfterRemoveScript) {
+                Remove-Item -erroraction 'silentlycontinue' $AfterScriptInfo.AfterRemoveScript
```

Comment:
-Force maybe?

Reply:
There will be a follow-up PR.  I can address these issue in that PR.